test/bgoogle.spec.ts

```diff
@@ -5,11 +5,11 @@ const baseUrl = 'http://www.google.com';
 describe('Given a SDET learning protractor', () => {
   describe('when open Google Page', () => {
     beforeEach(async () => {
-      browser.waitForAngularEnabled(false);
       await browser.get(baseUrl);
     });
     it('then should have a title', async () => {
-      expect(await browser.driver.getTitle()).toEqual('Google');
+      const title = (await browser.driver.getTitle()).toString();
```

Comment:
If you remove this changes, the test pass without needing to change the name of the fail or does it still fails?